Swift.md

```diff
@@ -2,9 +2,9 @@
 
 ## Sobre Swift
 
-[*Swift*](https://es.wikipedia.org/wiki/Swift_(lenguaje_de_programación)) es una nueva lenguaje de programmacion para applicaciones de iOS, OS X, watchOS, y tvOS que estan cimentados en lo mejor de [*C*](https://es.wikipedia.org/wiki/C_(lenguaje_de_programación)) y [*Objective-C*](https://es.wikipedia.org/wiki/Objective-C), sin las restricciones de compatibilidad con C. Swift adopta patrones de programmacion seguros y añade caracterisitacas modernas para hacer programacion mas facil, mas flexible, y mas divertido. La implementacion nueva y limpia de Swift, apoyado por las librerias maduras de [*Cocoa*](https://es.wikipedia.org/wiki/Cocoa_(informática)) y [*Cocoa Touch*](https://es.wikipedia.org/wiki/Cocoa_Touch), es una oportunidad para reimaginar como funciona el desarollo de Software.
```

Comment:
Spanish Wiki document of "framework" 

https://es.wikipedia.org/wiki/Framework